src/test/java/teammates/test/cases/webapi/EnrollStudentsActionTest.java

```diff
@@ -63,7 +95,8 @@ public void testExecute_withExistingStudent_shouldBeUpdatedToDatabase() {
         List<StudentData> enrolledStudents = executeActionAndReturnResults(courseId, req);
 
         assertEquals(1, enrolledStudents.size());
-        verifyStudentInDatabase(studentToUpdate, enrolledStudents.get(0).getCourseId(), enrolledStudents.get(0).getEmail());
+        verifyStudentInDatabase(
```

Comment:
Why add a new line here? Does this line contain too many characters?

Reply:
My bad. I thought the max char limit was 120. Reverted the change.
Anyway, yes, already manually tested. 

Thanks for reviewing!